script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
+        print(json.dumps(json.loads(json_file.read()), indent=2))
+
+def action_yaml(args):
```

Comment:
It is recommended to use the `safe_load` and the context manager for file handling here as well for consistency and resource management.
```suggestion
    with open('scripts/example_data.yaml', 'r') as yaml_file:
        data = yaml.safe_load(yaml_file)
        print(yaml.dump(data, indent=2))
```